.github/workflows/deploy.yml

```diff
@@ -82,9 +83,13 @@ jobs:
         with:
           name: test_results
           path: tests/nightwatch/tests_output
-      - name: Slack notification
-        if: env.SLACK_WEBHOOK_URL != '' && contains(fromJson('["develop", "main", "master", "impl", "val", "prod", "production"]'), env.branch_name) && always ()
-        uses: ravsamhq/notify-slack-action@master
-        with:
-          status: ${{ job.status }}
-          notify_when: "failure"
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
```

Comment:
Note the change to the 'uses:' key... this change uses a different github action to make slack notifications.  This new slack notifier action is docker based and available to pull/download, whereas the previous action was docker based but needed to be built each time.  Moving to this action saves 15 seconds per build, sometimes less sometimes more.

There's no change to the logic around "when" to notify.  Workflows for the listed branches will broadcast slack notifications on failure.